server/graphql/v2/input/TierUpdateInput.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  GraphQLBoolean,
+  GraphQLInputObjectType,
+  GraphQLInt,
+  GraphQLList,
+  GraphQLNonNull,
+  GraphQLString,
+} from 'graphql';
+
+import { TierAmountType, TierType } from '../enum';
+import { TierFrequency } from '../enum/TierFrequency';
+
+import { AmountInput } from './AmountInput';
+
+export const TierUpdateInput = new GraphQLInputObjectType({
+  name: 'TierUpdateInput',
+  fields: () => ({
+    id: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: 'The public id identifying the tier (ie: dgm9bnk8-0437xqry-ejpvzeol-jdayw5re)',
+    },
+    amount: {
+      type: new GraphQLNonNull(AmountInput),
+    },
+    name: {
+      type: new GraphQLNonNull(GraphQLString),
+    },
+    description: {
+      type: GraphQLString,
+    },
+    button: {
+      type: GraphQLString,
+    },
+    goal: {
+      type: AmountInput,
+    },
+    type: {
+      type: new GraphQLNonNull(TierType),
+    },
+    amountType: {
+      type: new GraphQLNonNull(TierAmountType),
+    },
+    frequency: {
+      type: new GraphQLNonNull(TierFrequency),
+    },
+    presets: {
+      type: new GraphQLList(GraphQLInt),
```

Comment:
This will require a small change in create/edit mutations resolvers

```suggestion
      type: new GraphQLList(new GraphQLNonNull(AmountInput)),
```